src/index.js

```diff
@@ -0,0 +1,362 @@
+  if (!target) return;
+  target.childNodes.forEach((ch) => {
+    if (!ch.classList.contains('hide')) {
+      key = ch.textContent;
+    }
+  });
+
+  if (target.getAttribute('data-code') !== 'CapsLock') {
+    target.classList.add('pressed');
+  } else {
+    target.classList.toggle('pressed');
+  }
+
+  const textarea = document.querySelector('#textarea');
+  if (!target.classList.contains('button_special')) {
+    let pozition = textarea.selectionStart;
+    const text = textarea.value;
+    const arr = text.split('');
+    arr.splice(pozition, 0, key);
+    textarea.value = arr.join('');
+    pozition += 1;
+    textarea.selectionStart = pozition;
+    textarea.selectionEnd = pozition;
+  } else {
+    switch (key) {
+      case 'Backspace': {
+        let pozition = textarea.selectionStart;
+        const text = textarea.value;
+        const arr = text.split('');
+        arr.splice(pozition - 1, 1);
+        textarea.value = arr.join('');
+        pozition -= 1;
+        textarea.selectionStart = pozition;
+        textarea.selectionEnd = pozition;
+        break;
+      }
+      case 'Tab': {
+        let pozition = textarea.selectionStart;
+        const text = textarea.value;
+        const arr = text.split('');
+        arr.splice(pozition, 0, '\t');
+        textarea.value = arr.join('');
+        pozition += 1;
+        textarea.selectionStart = pozition;
+        textarea.selectionEnd = pozition;
+        break;
+      }
+      case 'Delete': {
+        const pozition = textarea.selectionStart;
+        const text = textarea.value;
+        const arr = text.split('');
+        arr.splice(pozition, 1);
+        textarea.value = arr.join('');
+        textarea.selectionStart = pozition;
+        textarea.selectionEnd = pozition;
+        break;
+      }
+      case 'Enter': {
+        let pozition = textarea.selectionStart;
+        const text = textarea.value;
+        const arr = text.split('');
+        arr.splice(pozition, 0, '\n');
+        textarea.value = arr.join('');
+        pozition += 1;
+        textarea.selectionStart = pozition;
+        textarea.selectionEnd = pozition;
+        break;
+      }
+      case 'Shift':
+        if (localStorage.getItem('capslockPressed') === 'false') {
+          pressShift();
+        } else {
+          unpressShift();
+        }
+        break;
+      case 'Alt':
+        if (e.shiftKey) {
+          changeLanguage(e);
+          document.querySelectorAll('.button').forEach((btn) => btn.addEventListener('mousedown', mouseKeyDown));
+          document.querySelectorAll('.button').forEach((btn) => btn.addEventListener('mouseup', mouseKeyUp));
+          document.querySelectorAll('.button').forEach((btn) => btn.addEventListener('mouseout', mouseKeyOut));
+        }
+        break;
+      default:
+        break;
+    }
+  }
+};
+// keyboard events..
+const doSpecialKey = (e) => {
+  const textarea = document.querySelector('#textarea');
+  switch (e.key) {
+    case 'Backspace': {
```

Comment:
The same here. The logic are quite similar. You can use the same function as you create above